build/base/entrypoint.sh

```diff
@@ -26,7 +26,6 @@ function resolve_host() {
 }
 
 if [ "$K_MPI_JOB_ROLE" == "launcher" ]; then
-  resolve_host "$HOSTNAME"
```

Comment:
I don't think this call is necessary, is there are reason we need to resolve the launcher hostname?

Reply:
Yes, if the worker's can't resolve the launcher's hostname, they fail.

Reply:
Sure but that's something that the workers should deal with not the launcher trying to resolve itself. I'll add it back in case

Reply:
When I last tried, there were no retries in the Intel MPI implementation, so the pods would fail.
If the user configures `restartPolicy: Never`, then those jobs would always fail.

Waiting for the launcher to be resolvable is just more user friendly.